source/js/jquery-sortable.js

```diff
@@ -401,10 +402,11 @@
       ) >= this.options.distance)
     },
     getPointer: function(e) {
-      var o = e.originalEvent || e.originalEvent.touches && e.originalEvent.touches[0]
+      var o = e.originalEvent,
+        t = (e.originalEvent.touches && e.originalEvent.touches[0]) || {};
```

Comment:
If `e` stands for "event" and `o` stands for "originalEvent", then `t` stands for "touch".